docs/book/src/forc/manifest_reference.md

```diff
@@ -41,8 +42,64 @@ entry = "main.sw"
 organization = "Fuel_Labs"
 license = "Apache-2.0"
 name = "wallet_contract"
+
+[project.metadata]
+indexing = { namespace = "counter-contract", schema_path = "out/release/counter-contract-abi.json" }
 ```
 
+### Metadata Section in Forc.toml
```

Comment:
Putting this in backticks should satiate the spellchecker 

```suggestion
### Metadata Section in `Forc.toml`
```

Otherwise you can add "toml" to `spell-check-custom-words.txt`